packages/wrangler/src/__tests__/init.test.ts

```diff
@@ -2810,6 +2810,21 @@ describe("init", () => {
 	});
 });
 
+function getDefaultBranchName() {
+	try {
+		const { stdout: defaultBranchName } = execaSync("git", [
+			"config",
+			"--get",
+			"init.defaultBranch",
+		]);
+
+		return defaultBranchName;
+	} catch {
+		// ew
+		return "master";
```

Comment:
Should this not be `"main"`?

Reply:
Oh!! Yeah... why is that master again?

Reply:
That totally went over my head... 😅

Reply:
Because that's why CI was failing ...I guess we could do a more involved check...one second

Reply:
https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell#:~:text=The%20default%20branch%20name%20in,the%20last%20commit%20you%20made.